packages/boxel/types/ember-truth-helpers/helpers/not.d.ts

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import Helper from '@ember/component/helper';
+
+// NOTE: These types are somewhat imperfect.
+// For instance, the limit at 5 is arbitrary. Also, the actual helpers
+// use JavaScript truthiness along with special handling for empty arrays
+// and objects with an `isTruthy` method. In these cases, we just won't
+// narrow as much as we potentially could.
```

Comment:
Was this meant to be in `or`?

Reply:
Seems likely. I copied and pasted this from https://github.com/Gavant/glint-template-types/blob/main/types/ember-truth-helpers/not.d.ts

Reply:
Fixed.